### PR TITLE
feat: support bedrock embedding model provider

### LINKED ISSUE #379: llm: support bedrock llm & embedding

### PR DESCRIPTION
**Pull request by jrj5423**
Close #379

### CODE REVIEW

File: frontend/app/src/pages/docs/embedding-model.mdx [hunk 1]
```diff
@@ -90,6 +90,30 @@ Cohere provides industry-leading large language models (LLMs) and RAG capabiliti
...
+
+Amazon Bedrock is a fully managed foundation models service that provides a range of large language models and embedding models.
+
+**Supported Models**:
+
+| Embedding Model                 | Vector Dimensions | Max Tokens |
+| ------------------------------- | ----------------- | ---------- |
+| `amazon.titan-embed-text-v2:0`  | 1024              | 8192       |
+| `amazon.titan-embed-text-v1`    | 1536              | 8192       |
+| `amazon.titan-embed-g1-text-02` | 1536              | 8192       |
```
Comment (sykp241095) [inline frontend/app/src/pages/docs/embedding-model.mdx:103]: It seems AWS provides more models than these three?

For example cohere:xxxxx

Comment (634750802) [reply frontend/app/src/pages/docs/embedding-model.mdx:103]: I suggest to document featured models and add a *view more* link to provider's document page.

**Comment by Mini256** [comment]
Failed to create embedding model with bedrock provider. It seems some error on frontend side. cc: @634750802 

image

**Comment by 634750802** [comment]
> Failed to create embedding model with bedrock provider. It seems some error on frontend side. cc: @634750802
> 
> image

resolved in [f72c2e](https://github.com/pingcap/autoflow/commit/f72c2ecc8a7c33a5ee869a6d338bc4e7a83dc13d)

File: frontend/app/src/pages/docs/embedding-model.mdx [hunk 2]
```diff
@@ -109,6 +109,32 @@ Cohere provides industry-leading large language models (LLMs) and RAG capabiliti
 
 For more information, see the [Cohere Embed documentation](https://docs.cohere.com/docs/cohere-embed).
 
+### Amazon Bedrock
+
+Amazon Bedrock is a fully managed foundation models service that provides a range of large language models and embedding models.
+
+**Supported Models**:
```
Comment (Mini256) [inline frontend/app/src/pages/docs/embedding-model.mdx:116]:
```suggestion
**Featured Models**:
```

File: frontend/app/src/pages/docs/embedding-model.mdx [hunk 3]
```diff
@@ -109,6 +109,32 @@ Cohere provides industry-leading large language models (LLMs) and RAG capabiliti
...
+
+| Embedding Model                 | Vector Dimensions | Max Tokens |
+| ------------------------------- | ----------------- | ---------- |
+| `amazon.titan-embed-text-v2:0`  | 1024              | 8192       |
+| `amazon.titan-embed-text-v1`    | 1536              | 8192       |
+| `amazon.titan-embed-g1-text-02` | 1536              | 8192       |
+| `cohere.embed-english-v3`       | 1024              | 512        |
+| `cohere.embed-multilingual-v3`  | 1024              | 512        |
```
Comment (Mini256) [inline frontend/app/src/pages/docs/embedding-model.mdx:124]:
```suggestion
| `cohere.embed-multilingual-v3`  | 1024              | 512        |

To check all embbeding models supported by Bedrock, go to [Bedrock console](https://console.aws.amazon.com/bedrock).
```

File: frontend/app/src/pages/docs/embedding-model.mdx (hunk 1 above)
Comment (Mini256) [reply frontend/app/src/pages/docs/embedding-model.mdx:122]: Done!

**Review by Mini256 — APPROVED** [review]
LGTM, nice work 🚀

**Pull request merged**